cylc/sphinx_ext/__init__.py

```diff
@@ -23,7 +23,7 @@
 from shutil import copyfile
 
 
-__version__ = '1.0.0'
+__version__ = '1.2.0'
```

Comment:
Why was it 1.0.0 on here when the latest version on PyPI is 1.1.0, out of interest?

Reply:
It didn't get bumped with the last release.

Reply:
Oh, I'm surprised PyPI didn't reject it